src/main/java/seedu/address/model/person/Gender.java

```diff
@@ -77,7 +82,7 @@ public static boolean isValidGender(String test) {
 
     @Override
     public String toString() {
-        return value.toString();
+        return IsGenderSet ? value.toString() : "";
     }
```

Comment:
I think the title of this PR should be 'display "empty string" when no gender specified'?

Reply:
tbh the capital I is glaring strongly at me also, WHY ISNT TRAVIS BITCHING ABOUT THIS?

Reply:
Nice observation.